common/views/components/DefaultPageLayout/DefaultPageLayout.js

```diff
@@ -259,10 +251,11 @@ const DefaultPageLayout = ({
       <div id='main' className='main' role='main'>
         {children}
       </div>
-      <Footer
+      {/* <Footer
         openingHoursId='footer'
-        placesOpeningHours={placesOpeningHours}
-        upcomingExceptionalOpeningPeriods={upcomingExceptionalOpeningPeriods(exceptionalPeriods)} />
+        placesOpeningHours={[]/* TODO from getCollectionOpeningTimes }
+        upcomingExceptionalOpeningPeriods={[]/* TODO from getCollectionOpeningTimes } />
+      */}
```

Comment:
Could we not add this regression?

Reply:
Was going to put the fix for this in another PR; this ones getting quite large